lib/IR/GraphScheduler.cpp

```diff
@@ -14,206 +14,166 @@
  * limitations under the License.
  */
 #define DEBUG_TYPE "graph-scheduler"
+
+#include "GraphScheduler.h"
+
 #include "glow/Graph/Graph.h"
 #include "glow/Graph/Nodes.h"
 #include "glow/Graph/Utils.h"
 #include "glow/IR/IR.h"
 #include "glow/Support/Debug.h"
 
+#include "llvm/ADT/ArrayRef.h"
```

Comment:
@SplitInfinity In your previous comments you mentioned that ArrayRef, SmallVector and a few other headers are not used anywhere. Why don't you remove them then?